prow/config.yaml

```diff
@@ -6361,6 +6361,12 @@ periodics:
       env:
       - name: TEST_TMPDIR
         value: /root/.cache/bazel
+      # so we can use the right cache id
+      # TODO(bentheelder): switch to kubernetes_execute_bazel and consider dropping this
+      - name: REPO_OWNER
+        value: kubernetes
+      - name: REPO_NAME
+        value: kubernetes
```

Comment:
hummmm, maybe we should scan `--repo` flag and set those envs instead?

Reply:
how? in prow?

Reply:
bootstrap?

Reply:
In the future this will be handled by making checkout first class, for the moment though we need this to unblock periodic job caching.

If you mean in bootstrap, I want to depend *less* on magical bootstrap behavior and these args are normally set by prow for presubmit/postsubmit